plugins/serializers/wavefront/wavefront_test.go

```diff
@@ -131,7 +131,7 @@ func TestFormatMetricPoint(t *testing.T) {
 		},
 	}
 
-	s := WavefrontSerializer{}
+	s := WavefrontSerializer{ConvertPaths: true}
```

Comment:
That's exactly the point mentioned above. All current users would need to do the same in their config. Please keep existing tests unchanged to ensure backward compatibility!

Reply:
Good point!
Let me fix that.